src/pages/projects/ProjectJSON.tsx

```diff
@@ -1,30 +1,32 @@
 export default {
```

Comment:
🌱 this should be type-checked

Reply:
Yep @54skyxenon do you have time to fix these?

Reply:
Yeah sure thing